private/dependency_tree_parser.bzl

```diff
@@ -119,8 +119,9 @@ def _generate_imports(repository_ctx, dep_tree, explicit_artifacts, neverlink_ar
                 import_rule = "aar_import"
             else:
                 fail("Unsupported packaging type: " + packaging)
-            if repository_ctx.attr.jetify:
+            if repository_ctx.attr.jetify and _simple_coord_matches_pattern(simple_coord, repository_ctx.attr.jetify_include_patterns):
```

Comment:
I can't see context on a phone. Is this run in a loop? Of so then we may need to cache things in the function so we don't recalculate const things.